sdk/identity/Azure.Identity/samples/TokenCache.md

```diff
@@ -18,6 +18,10 @@ The in-memory token cache provided by the Azure Identity library is thread-safe.
 
 **Note:** When Azure Identity library credentials are used with Azure service libraries (for example, Azure Blob Storage), the in-memory token caching is active in the `HttpPipeline` layer as well. All `TokenCredential` implementations are supported there, including custom implementations external to the Azure Identity library.
 
+#### Disable caching
+
+As there are many levels of cache, it's not possible to disable in-memory caching. However, the in-memory cache may be cleared by creating a new credential instance.
```

Comment:
I think the exception here is when using DefaultAzureCredential without any options. That ends up being a static singleton.